### ISSUE TITLE
UWqidAS

### ISSUE THREAD

**Issue by sthompson258** [AnKing-Memberships/ankiweb-placeholder#53]
#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [ ] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [ ] I've checked if anyone else asked this question before by looking through the issue reports.

#### What Anki version are you using?

*Please open Anki, go to Help → About (windows) or Anki → About Anki (mac) to find the version. Edit the value between the backticks below

```
2.1.##
```

#### Your question

*A clear and concise question about the add-on.*

**Comment by sthompson258** [comment]
Hey I bought a subscription about a month ago so I could download this and recently noticed the add on isn't working. Is the only way for me to get it working again to buy another subscription and redownload?

**Comment by AnKingMed** [comment]
If it says UWQidAS then it's just the placeholder. Did you originally download from the membership and it was working?  And it stopped working but you don't have a membership anymore?

**Comment by sthompson258** [comment]
I bought membership on 3/28/23 and downloaded from there and had it
working. I noticed today that the UWorld tab wasn't showing up in anki and
I looked at my add-ons list and saw what I assumed was the right add-on but
it still didn't work. I went to the site with the placeholder to try and
figure out what I could do to fix it, I downloaded the placeholder to see
if that would do anything (which of course it didn't) and now it looks like
I don't have the original add-on at all now.
